### DISCUSSION TITLE
Anyone facing similar issues ?

### DISCUSSION THREAD

**Discussion by Akhil-Tony** [github/india#245] [Students]
[anyone] 
![Screenshot (14)](https://user-images.githubusercontent.com/72643926/136154001-4072e9ce-c434-47b9-9976-f70fa16e4df7.png)
?

**Comment by Aditya-Bhate** [comment]
me

**Comment by ghost** [comment]
Yes!

**Reply by crazyuploader** [reply]
Simply logging out, and logging back in, solved the problem for me. Try that, might start working again.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
